README.md

```diff
@@ -136,15 +144,15 @@ def get_provider_info():
     }
 ```
 
-Once you define the entrypoint, you can leverage Airflow's native features to expose custom connection types in the Airflow UI and additional links to relevant pages of documentation and information.
+Once you define the entrypoint, you can use native Airflow features to expose custom connection types in the Airflow UI, as well as additional links to relevant documentation.
 
-### Adding Custom Connections
+### Adding Custom Connection Forms
 
-Airflow allows for custom connection forms through discoverable hooks. Below is an example of a custom connection form for the Fivetran provider.
+Airflow enables custom connection forms through discoverable hooks. The following is an example of a custom connection form for the Fivetran provider:
 
 <img src="https://user-images.githubusercontent.com/63181127/112921463-d07b2880-90d8-11eb-871b-fc4e1c6cade9.png" width="600" />
 
-Add code to the hook class to initiate a discoverable hook and create a custom connection form. The following is an example of this code.
+Add code to the hook class to initiate a discoverable hook and create a custom connection form. The following code defines a hook and a custom connection form:
 
 ```python
 class ExampleHook(BaseHook):
```

Comment:
If it's possible, it'd be great to have this code sample reflect either the example above, have the screen shot reflect this code sample, or have this code sample reflect the get_provider_info template.

Reply:
Agreed. Will provide better detail.